scripts/executors/invalidate-just-cache.js

```diff
@@ -0,0 +1,9 @@
+const fs = require('fs');
```

Comment:
removed `fs-extra` as `fs` is enables same behaviours

this was moved rather than created. whats wrong with you git ? :D